src/frontend/src/icp/components/fee/BitcoinKYTFee.svelte

```diff
@@ -1,29 +1,28 @@
 <script lang="ts">
 	import { slide } from 'svelte/transition';
 	import { ckBtcMinterInfoStore } from '$icp/stores/ckbtc.store';
-	import { token, tokenId } from '$lib/derived/token.derived';
+	import { tokenId, tokenWithFallback } from '$lib/derived/token.derived';
 	import { nonNullish } from '@dfinity/utils';
 	import { isTokenCkBtcLedger } from '$icp/utils/ic-send.utils';
 	import type { IcToken } from '$icp/types/ic';
 	import type { NetworkId } from '$lib/types/network';
 	import { formatToken } from '$lib/utils/format.utils';
 	import { BigNumber } from '@ethersproject/bignumber';
 	import Value from '$lib/components/ui/Value.svelte';
-
 	import { BTC_DECIMALS } from '$env/tokens.btc.env';
 	import { i18n } from '$lib/stores/i18n.store';
 	import { isNetworkIdBitcoin } from '$lib/utils/network.utils';
 
 	export let networkId: NetworkId | undefined = undefined;
 
 	let ckBTC = false;
-	$: ckBTC = isTokenCkBtcLedger($token as IcToken);
+	$: ckBTC = isTokenCkBtcLedger($tokenWithFallback as IcToken);
 
 	let btcNetwork = false;
 	$: btcNetwork = isNetworkIdBitcoin(networkId);
 
 	let kytFee: bigint | undefined = undefined;
-	$: kytFee = $ckBtcMinterInfoStore?.[$tokenId]?.data.kyt_fee;
+	$: kytFee = nonNullish($tokenId) ? $ckBtcMinterInfoStore?.[$tokenId]?.data.kyt_fee : undefined;
```

Comment:
I am not 100% sure (maybe 90%), but I think that it would return `undefined` either way, if you pass `$tokenId` nullish here

`$: kytFee = $ckBtcMinterInfoStore?.[$tokenId]?.data.kyt_fee;`

you already tried?

Reply:
Yeah, I wish too.

> Svelte: Type null cannot be used as an index type.
> Svelte: Type undefined cannot be used as an index type.

<img width="1470" alt="Capture d’écran 2024-06-17 à 16 36 36" src="https://github.com/dfinity/oisy-wallet/assets/16886711/c8dabad3-53b5-4d24-9226-48889a48ce71">